script/metalsmith/breadcrumbTransformer.js

```diff
@@ -0,0 +1,22 @@
+// const jsdom = require('jsdom');
+const cheerio = require('cheerio');
+const path = require('path');
+
+module.exports = (files, metalsmith, done) => {
+  Object.keys(files).forEach(file => {
+    if (path.extname(file) !== '.html') return;
+
+    const data = files[file];
+    const $ = cheerio.load(data.contents.toString());
+
+    $('#va-breadcrumbs').each(() => {
+      $.root().append(
+        '<script type="text/javascript" src="/js/breadcrumbs/mobile-breadcrumb.js"></script>'
```

Comment:
I wonder if it might be better to inline this script. It's a little long for that, but we are starting to stack up small external JS scripts, which is pushing back the time to first byte on our main bundles.